README.md

```diff
@@ -1624,6 +1624,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Roberto Jr. Restituyo](https://github.com/juniorrestituyo)
   - [Robertus Bagaskara](https://github.com/robertusbagaskara)
   - [Rodrigo Avila](https://github.com/rodrigo398)
+  - [Rohan Sharma](https://github.com/rohansx)
```

Comment:
Please add your name in line 1630 after Rohan Nair.
Let us know if you need any help with that. 🙂

Reply:
Done. Please review it again @Panquesito7